storage/mock/src/main/java/com/emc/mongoose/storage/mock/Main.java

```diff
@@ -54,14 +54,26 @@ public static void main(final String[] args)
 		final Config.ItemConfig itemConfig = config.getItemConfig();
 		final StorageMockFactory storageMockFactory =
 			new StorageMockFactory(storageConfig, loadConfig, itemConfig);
-		try(final Daemon storageMock = storageMockFactory.newNagaina()) {
-			storageMock.start();
-			try {
-				storageMock.await();
-			} catch(final InterruptedException ignored) {
+		if(storageConfig.getNode()) {
```

Comment:
1. this should be storage.mock.remoteNode flag
2. I don't see the defaults.json according changes neither defaults-schema.json changes